src/app/services/actions/query-action-creators.ts

```diff
@@ -118,3 +119,24 @@ async function createHistory(response: Response, respHeaders: any, query: IQuery
   return result;
 }
 
+export function internalQuery(query: IQuery): Function {
```

Comment:
i think this function might need a more descriptive name, what does `internalQuery` do?

Reply:
2218394bef01c8cec6fa541a61a911a839926c70

Reply:
noice